src/Sylius/Behat/Context/Api/Admin/ManagingCatalogPromotionsContext.php

```diff
@@ -101,6 +101,26 @@ public function iSpecifyItsAsIn(string $field, string $value, string $localeCode
         $this->client->updateRequestData($data);
     }
 
+    /**
+     * @When I disable this catalog promotion
+     */
+    public function iDisableCatalogPromotion(): void
+    {
+        $data = ['enabled' => false];
+
+        $this->client->updateRequestData($data);
+    }
```

Comment:
```suggestion
    public function iDisableCatalogPromotion(): void
    {
        $this->client->updateRequestData('enabled' => false);
    }
```

Wdyt?
Possibly it might be in [] 😄